src/reducer/expense.js

```diff
@@ -21,19 +21,20 @@ export default (state = initialState, action) => {
       return { ...state, [payload.categoryId]: [...expenses, payload] };
 
     case 'EXPENSE_DESTROY':
+      if (!payload.expense){payload.expense = {id: payload.id}};

```

Comment:
Make sure to get a space between the parens and curlys if you are going to one line an if statement.